src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -235,6 +235,23 @@ This instrumentation automatically sets Activity Status to Error if an unhandled
 exception is thrown. Additionally, `RecordException` feature may be turned on,
 to store the exception to the Activity itself as ActivityEvent.
 
+## Activity Duration and http.server.request.duration metric calculation
+
+`Activity.Duration` and `http.server.request.duration` values represents the
+time used to handle an inbound HTTP request as measured at the hosting layer of
+ASP.NET Core. The time measurement starts once the underlying web host has:
+
+* Sufficiently parsed the HTTP request headers on the inbound network stream to
+  identify the new request.
+* Initialized the context data structures such as the
+  [HttpContext](https://learn.microsoft.com/dotnet/api/microsoft.aspnetcore.http.httpcontext).
```

Comment:
does it also include the time to read requestpayload like in a POST request payload?

Reply:
As per my understanding, Yes. The request payload is not read until it is needed. I will get this verified with someone from the ASP.NET Core team https://github.com/open-telemetry/opentelemetry-dotnet/issues/5096

Reply:
I confirmed this with the .NET team. The time to read request payload is included. I would see if we can get this information added upstream and then add it here as well, will keep the issue opened for that.